test-runner.sh

```diff
@@ -4,6 +4,7 @@ set -o pipefail
 
 get_changes ()
 {
+	git remote remove current && \
```

Comment:
I found I needed this in order to run the tests more than once. Is it okay if it lands with this PR?